docker/gremlin-test-server/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN chmod 755 /opt/docker-entrypoint.sh
 # Installing dos2unix to avoid errors in running the entrypoint script on Windows machines where their
 # carriage return is \r\n instead of the \n needed for linux/unix containers
 RUN apk update && apk add dos2unix
-RUN dos2unix /opt/docker-entrypoint.sh && apk del dos2unix
+RUN dos2unix /opt/docker-entrypoint.sh /opt/gremlin-server/bin/gremlin-server.sh && apk del dos2unix
```

Comment:
What exactly does this change fix?

Reply:
windows-like EOF:
`/opt/docker-entrypoint.sh: /opt/gremlin-server/bin/gremlin-server.sh: /bin/bash^M: bad interpreter: No such file or directory`

Reply:
oh gotcha